src/cloudant/design_document.py

```diff
@@ -243,3 +207,128 @@ def info(self):
         GET databasename/_design/{ddoc}/_info
         """
         raise NotImplementedError("_info not yet implemented")
+
+class CloudantDesignDocument(DesignDocument):
+    """
+    Encapsulates a Cloudant version of a
+    :class:`~cloudant.design_document.DesignDocument`.
+    A CloudantDesignDocument object is instantiated with a reference
+    to a Cloudant database and provides an API to query index management,
+    list and show functions, etc.
+
+    Note:  Currently only the query view management API exists.  Remaining
+    Cloudant design document functionality will be added later.
+
+    :param database: A ``CloudantDatabase`` instance used by the
+        CloudantDesignDocument.
+    :param str document_id: Optional document id.  If provided and does not
+        start with ``_design/``, it will be prepended with ``_design/``.
+    """
+    def __init__(self, database, document_id=None):
+        super(CloudantDesignDocument, self).__init__(database, document_id)
```

Comment:
I don't think you need the `__init__` constructor here since you're not performing any additional constructor logic.  The `__init__` should automatically inherit its behavior from DesignDocument.